javascript/ql/lib/semmle/javascript/frameworks/Tanstack.qll

```diff
@@ -0,0 +1,26 @@
+/**
+ * Provides classes and predicates modeling the Tanstack/react-query library.
+ */
+
+private import javascript
+
+/**
+ * An additional flow step that propagates data from the return value of the query function,
+ * defined in a useQuery call from the '@tanstack/react-query' module, to the 'data' property.
+ */
+class TanstackStep extends DataFlow::AdditionalFlowStep {
```

Comment:
```suggestion
private class TanstackStep extends DataFlow::AdditionalFlowStep {
```

Reply:
Since this file is `import`'ed from `javascript.qll` anything non-private we put here will show up in the top-level scope pretty much everywhere. We'd like to avoid that so we tend to make things private or wrap them in a module.